OJP/OJP_Fare.xsd

```diff
@@ -142,6 +137,11 @@
 				</xs:element>
 			</xs:choice>
 		</xs:sequence>
+		<xs:attribute name="id" type="ObjectIdType">
```

Comment:
I know there was a reluctance to use attributes in the OJP model, I don't know why exactly, IIRC due to be able to map it easier to other data structures like JSON? @pvgrumbkow

Reply:
Yes, Werner confirmed that in TRIAS the mapping, besides the consistency with SIRI, was the reason to do without attributes in most cases.

Reply:
@herlitze @sgrossberndt Next time pls state so during the meeting (we discussed this two times and I specifically asked if I should go into that direction). Would have saved me the work. For JSON: Our guys would have prefered the attributes. I will add this to the discussion list for the next meeting.

Reply:
Stephan: Renaming, but element and not attribute. We want to stay consistent with Siri.

Reply:
adapted